documentation/asciidoc/topics/proc_creating_service_monitor.adoc

```diff
@@ -1,32 +1,30 @@
-[id='creating_service_monitor-{context}']
+[id='creating-service-monitor_{context}']
 = Creating a Prometheus Service Monitor
-Define a service monitor instances that configures Prometheus to monitor your
-{brandname} cluster.
+Define service monitor instances that configures Prometheus to monitor your {brandname} cluster.
 
 .Prerequisites
 
+* Enable monitoring for your {openshiftshort} project.
```

Comment:
This is very vague. Do you mean openshift user-workload-monitoring? Or just installing Grafan and prometheus operators?

Reply:
I've reworded this. I think this initially comes from several OpenShift versions ago when it wasn't really clear if you could enable monitoring in the default stack or not.